Watchman.Discord/Areas/Users/Controllers/UsersController.cs

```diff
@@ -97,9 +104,16 @@ public void RemoveRole(string message, Dictionary<string, IDiscordContext> conte
             var serverContext = (DiscordServerContext)contexts[nameof(DiscordServerContext)];
             var userService = new UserService();
             var serverRole = userService.GetRoleByName(commandRole, serverContext);
-            userService.RemoveRole(serverRole, userContext, serverContext);
 
-            messagesService.SendMessage($"Usunięto role {commandRole} użytkownikowi {userContext}");
+            try
+            {
+                userService.RemoveRole(serverRole, userContext, serverContext).Wait();
```

Comment:
jeśli to wyrzuca wyjątki, to może na wyższym poziomie dałoby się to zautomatyzować, robiąc po prostu tłumaczenie treści wyjątków 

a jeśli do każdej akcji jest inna klasa wyjątku to już super